### DISCUSSION TITLE
Player in command

### DISCUSSION THREAD

**Discussion by sqcred** [Minestom/Minestom#658] [Help] [answered]
How do i set the player argument in a command?

**Comment by GreatWyrm** [comment, marked as answer]
`ArgumentEntity players = ArgumentType.Entity("player").onlyPlayers(true);`

You will still get a ` List<Entity>` from `context.get(players).find(sender);`, but you can safely cast all entities in the list to Player.

**Reply by sqcred** [reply]
Ok thanks so much
